README.md

```diff
@@ -138,5 +166,11 @@ power encoding, and VP9 low power encoding require the kernel patch for GuC supp
 
 5. CNL: HEVC encoding does not support P frame.
 
+6. CFL: BRC functionalities requiring HuC for AVC low power encoding require 4.15 or later kernel to work.
```

Comment:
4.11 and 4.15 looks ok. I think we need to mention kernel boot parameter to access HuC related features. This parameter looks to be i915.enable_guc=2 (bitmask, loading GuC is a prerequisite for HuC).

Can you reformulate what "BRC functionalities requiring HuC" means? I would like to see something like:

> The following features require HuC FW support:
> * CBR, VBR, .... (all BRC modes, CQP, ... are not affected) modes of AVC low power encoding
> * other feature, if any
> * other feature, if any
>
> As of now HuC FW support is disable in Linux kernels by default. To enable it load the kernel with `i915.enable_guc=2` (bitmask, loading GuC is a prerequisite for HuC) parameter. Mind that HuC FW support presents in the following kernels for the specified platforms:
> * APL, KBL: starting from kernel 4.11
> * CFL: starting from kernel 4.15

Thus, I would group all the notes about HuC together. By the way, why you don't mention SKL?

CNL: I don't understand where I can get HuC FW! I don't see it in the git://anongit.freedesktop.org/drm/drm-firmware. Thus, maybe we need to add a note for CNL like:
> CBR, VBR, etc. modes of AVC low power encoding are not supported (HuC is not available for CNL)